data/net/minecraft/world/level/block/SculkBlock.mapping

```diff
@@ -1,6 +1,11 @@
 CLASS net/minecraft/world/level/block/SculkBlock
+	METHOD canPlaceGrowth (Lnet/minecraft/world/level/LevelAccessor;Lnet/minecraft/core/BlockPos;)Z
+		ARG 0 level
+		ARG 1 pos
+	METHOD getDecayPenalty (Lnet/minecraft/world/level/block/SculkSpreader;Lnet/minecraft/core/BlockPos;Lnet/minecraft/core/BlockPos;I)I
+		ARG 0 spreader
 	METHOD getRandomGrowthState (Lnet/minecraft/world/level/LevelAccessor;Lnet/minecraft/core/BlockPos;Lnet/minecraft/util/RandomSource;Z)Lnet/minecraft/world/level/block/state/BlockState;
 		ARG 1 level
 		ARG 2 pos
 		ARG 3 random
-		ARG 4 isWorldGeneration
+		ARG 4 canSummon
```

Comment:
Also investigation and explanation on this one.

Reply:
I think `canSummon` makes sense. The parameter is used to set the value of the can summon block state on a sculk shrieker.
```
BlockState $$4 = (BlockState)Blocks.SCULK_SHRIEKER.defaultBlockState().setValue(SculkShriekerBlock.CAN_SUMMON, canSummon);
```

Reply:
Is it not passed in as isWorldGeneration elsewhere?

Reply:
It is actually, I'll change it back.

Reply:
Nope not fixed.